paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -159,6 +159,11 @@ def paasta_start_or_stop(args, desired_state):
     else:
         clusters = list_clusters(service)
```

Comment:
hm, yeah, I kind of agree. I don't know what a sane default might be though, so we should probably require a cluster?